components/patient-data/ui/src/main/resources/PhenoTips/PatientConsent.xml

```diff
@@ -677,6 +677,9 @@
 .error_updating_consents {
   margin-top: 1em;
   margin-left: 2em;
+}
+.view-title-required-not-granted {
+  font-size: 1.15em;
```

Comment:
Small version looks cleaner to me 
@allasm wdyt? 
![small](https://cloud.githubusercontent.com/assets/4371097/19742272/494c33bc-9b92-11e6-85a1-43dfcf2aa91c.jpg)
![big](https://cloud.githubusercontent.com/assets/4371097/19742275/4a8a48f4-9b92-11e6-8fe4-69b3d96536e3.jpg)